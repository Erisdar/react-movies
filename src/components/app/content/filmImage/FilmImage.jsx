import React from 'react';
import './filmImage.css';

export default ({ logo, alt }) => (<img src={logo} alt={alt} className="film-item__film-image" />);

