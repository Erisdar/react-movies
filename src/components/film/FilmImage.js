import React from 'react';
import '../../styles/film/filmImage.css';

export default ({logo, alt}) => (
    <img src={require(`../../img/${logo}`)} alt={alt} className="film-logo" />
);