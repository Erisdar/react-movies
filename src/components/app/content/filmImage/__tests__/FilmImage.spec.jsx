import React from 'react';
import { shallow } from 'enzyme';
import FilmImage from '../FilmImage';

describe('Film image component', () => {
    it('should be render correctly', () => {
        const component = shallow(<FilmImage logo={'harry_potter.jpg'} alt={'HarryPotter'} />);
        expect(component).toMatchSnapshot();
    });
});