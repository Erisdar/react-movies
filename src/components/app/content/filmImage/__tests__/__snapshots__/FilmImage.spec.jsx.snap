// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Film image component should be render correctly 1`] = `
<img
  alt="HarryPotter"
  className="film-item__film-image"
  src="harry_potter.jpg"
/>
`;
